contracts/patriciaTree/Bits.sol

```diff
@@ -33,11 +33,11 @@ library Bits {
     require(self != 0, "colony-patricia-tree-zero-self");
     uint val = self;
     for (uint8 i = 128; i >= 1; i >>= 1) {
-      if (val & (ONE << i) - 1 == 0) { // ignore-swc-101 TODO: this one also only shows up if analysing PatricaTree.sol
+      if (val & ((ONE << i) - 1) == 0) {
+        // ignore-swc-101 TODO: this one also only shows up if analysing PatricaTree.sol
```

Comment:
Does moving the comment to the next line mess up the slither parsing? We may need to move it to the line above and add `ignore-next-line` or something

Reply:
Seemingly not, as the build is green?